automatminer/automl/keras_wrapper.py

```diff
@@ -0,0 +1,157 @@
+from __future__ import unicode_literals
```

Comment:
We are working in Python3.6+ exclusively. If I remember correctly, unicode_literals are standard in python3 and this import is not needed.